src/drive/web/modules/views/Public/usePublicFilesQuery.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect, useRef } from 'react'
+import { useClient } from 'cozy-client'
+import get from 'lodash/get'
+
+const statById = async (client, folderId, cursorToUse) => {
+  // Most stack routes are off-limit when we have a read-only token, so we use a simple GET to load the folder content.
+  // no query because we need to paginate the included files
+  const { included, links } = await client
+    .collection('io.cozy.files')
+    .statById(folderId, {
+      'page[cursor]': cursorToUse
+    })
+
+  const nextLink = get(links, 'next', '')
+  const nextLinkURL = new URL(`http://example.com${nextLink}`)
```

Comment:
What is this exemple.com?

Reply:
I've clarified this part of the code and added a comment.